docs/_source/reference/python/python_client.rst

```diff
@@ -32,4 +34,12 @@ Datasets
 --------
 
 .. automodule:: argilla.client.datasets
-   :members: DatasetForTextClassification, DatasetForTokenClassification, DatasetForText2Text, read_datasets, read_pandas
\ No newline at end of file
+   :members: DatasetForTextClassification, DatasetForTokenClassification, DatasetForText2Text, read_datasets, read_pandas
+
+.. _python ref feedbackdataset:
+
+FeedbackDataset
+--------
+
+.. automodule:: argilla.client.feedback
+   :members: FeedbackDataset, FeedbackDatasetConfig, RatingQuestion, TextQuestion, QuestionSchema, TextField, FieldSchema, OfflineFeedbackRecord, FeedbackRecord, create_feedback_dataset
```

Comment:
I guess we can skip `create_feedback_dataset` for the moment, since it's not included in the `__init__`